Engines/Wine/Verbs/vulkanSDK/script.js

```diff
@@ -20,18 +22,48 @@ Wine.prototype.vulkanSDK = function () {
         .get();
 
     this.run(setupFile, "/S");
+    this.wait();
+
+    var pathVulkanJSON = this.prefixDirectory() + "drive_c/windows/winevulkan.json";
+    touch(pathVulkanJSON);
+    var contentVulkanJSON = '{\n'                                                                  +
+                            '	"file_format_version": "1.0.0",\n'                                 +
+                            '	"ICD": {\n'                                                        +
+                            '		"library_path": "c:\\windows\\system32\\winevulkan.dll",\n'    +
+                            '		"api_version": "' + sdkVersion +'"\n'                          +
+                            '	}\n'                                                               +
+                            '}' ;
+
+    writeToFile(pathVulkanJSON, contentVulkanJSON);
+
+    var regeditFileContent32 =
+    "Windows Registry Editor Version 5.00\n"                        +
+    "\n"                                                      	    +
+    "[HKEY_LOCAL_MACHINE\\Software\\Khronos\\Vulkan\\Drivers]\n"    +
+    "\"C:\\\\Windows\\\\winevulkan.json\"=dword:00000000" ;
 
-    var patchVulkanJSON = this.prefixDirectory() + "drive_c/windows/winevulkan.json";
-    touch(patchVulkanJSON);
-    writeToFile(patchVulkanJSON, "{\n	\"file_format_version\": \"1.0.0\",\n	\"ICD\": {\n		\"library_path\": \"c:\windows\system32\winevulkan.dll\",\n		\"api_version\": \"1.1.92.1\"\n	}\n}");
-    this.run("reg", ["add", "HKLM\Software\Khronos\Vulkan\Drivers", "/v", "C:\Windows\winevulkan.json", "/t", "REG_DWORD", "/d", "00000000", "/f"], null, false, true);
+    this.regedit().patch(regeditFileContent32);
 
     if (this.architecture() == "amd64") {
-        this.run("reg", ["add", "HKLM\Software\Wow6432Node\Khronos\Vulkan\Drivers", "/v", "C:\Windows\winevulkan.json", "/t", "REG_DWORD", "/d", "00000000", "/f"], null, false, true);
+        var regeditFileContent64 =
+        "REGEDIT4\n"                                                                  +
+        "\n"                                                                          +
+        "[HKEY_LOCAL_MACHINE\\SOFTWARE\\Wow6432Node\\Khronos\\Vulkan\\Drivers\\]n"    +
+		"\"C:\\\\Windows\\\\winevulkan.json\"=dword:00000000" ;
+
+        this.regedit().patch(regeditFileContent64);
     }
+    
+    var message = "Please add the following keys to the regsitry after the instalation:\n\n"    +
+                  "[HKEY_LOCAL_MACHINE\\Software\\Khronos\\Vulkan\\Drivers]\n"                  +
+                  "\"C:\\\\Windows\\\\winevulkan.json\"=dword:00000000"                         +
+                  "\n\n"                                                            	        +
+                  "[HKEY_LOCAL_MACHINE\\SOFTWARE\\Wow6432Node\\Khronos\\Vulkan\\Drivers\\]n"    +
+                  "\"C:\\\\Windows\\\\winevulkan.json\"=dword:00000000" ;
 
-    return this;
+    this.wizard().message(message);

```

Comment:
I don't understand why showing this to the user

Reply:
Why it does not work? 
This is not acceptable to ask him to do it in terms of user experience

Reply:
Seems my comment did not get uploaded ^^.

Reply:
I've just seen it. This could not be a bug in wine

Reply:
I have just tested it and it is.

Reply:
The file must not be correctly formatted. Why would it not work just with this particular reg file?

Reply:
Look, you've left a trailing "n" character